src/mouse.js

```diff
@@ -4,7 +4,8 @@ var pointermap = dispatcher.pointermap;
 // radius around touchend that swallows mouse events
 var DEDUP_DIST = 25;
 
-var WHICH_TO_BUTTONS = [0, 1, 4, 2];
+// left, middle, right, back, forward
+var BUTTON_TO_BUTTONS = [1, 4, 2, 8, 16];
```

Comment:
Are you aware of any overview on hardware/UA-combinations that send more button-codes? The DOM3-Events draft specifies that
> Some pointing devices provide or simulate more button states, and values higher than 2 or lower than 0 MAY be used to represent such buttons.

and <0 would cause trouble with both versions, but I don't know if any system actually uses those button codes.

As the code mapping only has to be done for `mousedown` and `mouseup` the speed `pow` vs. `map` should not be a problem, I guess. But I'd like to wait for the result of the `which` vs. `button` discussion below before selecting a method (as [jquery's `which`](https://github.com/jquery/jquery/blob/master/src/event.js#L510) does not seem to support more buttons states either).